umap/static/umap/js/umap.js

```diff
@@ -192,16 +192,15 @@ L.U.Map.include({
       this
     )
 
-    let isDirty = false // global status
+    let isDirty = false // self status
     try {
       Object.defineProperty(this, 'isDirty', {
         get: function () {
-          return isDirty || this.dirty_datalayers.length
+          return isDirty
```

Comment:
Not sure why this check on `dirty_datalayers` was here before. As I can see, the `isDirty` status should always be set when a datalayer is made dirty…